src/Microsoft.CodeQuality.Analyzers/UnitTests/ApiDesignGuidelines/UsePropertiesWhereAppropriateTests.cs

```diff
@@ -4,12 +4,6 @@
 using Microsoft.CodeAnalysis.Testing;
 using Test.Utilities;
 using Xunit;
-using VerifyCS = Test.Utilities.CSharpCodeFixVerifier<
```

Comment:
Unused

Reply:
These should not be removed. They are only unused because the tests are written using the legacy test framework and still needed to be updated to use these. I added them to all the tests so it's easier for users to aid in this conversion (as separate pull requests).